CMakeLists.txt

```diff
@@ -202,6 +202,9 @@ elseif(FREEPLAYTECH_WAVESHARE32B)
 elseif(ADAFRUIT_HX8357D_PITFT)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DHX8357D -DADAFRUIT_HX8357D_PITFT")
 	message(STATUS "Targeting Adafruit 3.5 inch PiTFT with HX8357D")
+elseif (ADAFRUIT_ST7789_PITFT_114)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DADAFRUIT_ST7789_PITFT_114")

```

Comment:
Instead of duplicating in all places below `defined(ST7789) || defined(ADAFRUIT_ST7789_PITFT_114)`, let's instead add here

```
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DADAFRUIT_ST7789_PITFT_114 -DST7789")
```
since ADAFRUIT_ST7789_PITFT_114 uses ST7789. Then all ST7789 code blocks will automatically apply to ADAFRUIT_ST7789_PITFT_114.